integration-tests/main/pom.xml

```diff
@@ -23,6 +23,9 @@
 
         <!-- do not update this dependency, it is only used for testing -->
         <webjar.jquery-ui.version>1.13.0</webjar.jquery-ui.version>
+
+        <quarkus.config-tracking.enabled>true</quarkus.config-tracking.enabled>
+        <quarkus.native.container-build>true</quarkus.native.container-build>
```

Comment:
@rsvoboda why is this change necessary?

Reply:
This is required to make the Quarkus build goal cacheable:
https://github.com/gradle/develocity-build-config-samples/tree/main/quarkus-build-caching-extension

Reply:
Good, thanks for the pointer @jprinet